plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/holographicdisplays/FunnyHologramManager.java

```diff
@@ -0,0 +1,16 @@
+package net.dzikoysk.funnyguilds.feature.hooks.holographicdisplays;
+
+import net.dzikoysk.funnyguilds.guild.Guild;
+import panda.std.Option;
+
+public interface FunnyHologramManager {
```

Comment:
To samo co z FunnyHologram. Oraz putanie czy nazwa Manager jest dobra? By sie przydało wymyślić coś lepszego, Manager to dość ogólna nazwa.

Reply:
Service? `( ͡° ͜ʖ ͡°)`

Reply:
Less goooo